### DISCUSSION TITLE
New Jellyfin LXC - can't open web page

### DISCUSSION THREAD

**Discussion by di5cord20** [tteck/Proxmox#3589] [Help Needed]
I've installed a new Jellyfin LXC (wanted to convert existing privilaged version but ran into errors).  The LXC installs and is running.  I'm able to see the jellyfin version in the console of the LXC.  When I try to navigate to the url:8096 port, nothing happens (site can't be reached).  I've killed and re-created several time, same results.

Any ideas on how to troubleshoot or fix?

Note: my privilaged Jellyfin LXC is still up and running (and works).

**Comment by labob246** [comment]
I literally have been dealing with this same problem all day today.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/3592

**Comment by di5cord20** [comment]
That solved my probelm. Thanks tteck as always.
https://github.com/tteck/Proxmox/issues/3592#issuecomment-2309076120
